src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.nav {
+  display: flex;
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: #060b35;
+    font-size: 12px;
+    line-height: 60px;
```

Comment:
The `line-height` property is usually used to vertically center text within a container. If the height of your container is not fixed, or can change, this could potentially lead to issues. It's better to use flexbox properties for vertical centering.